.circleci/config.yml

```diff
@@ -48,6 +48,12 @@ jobs:
               else
                 mvn -B -s .circleci/settings.xml deploy
               fi
+            elif [[ -n "${DEPLOY_BRANCH}" ]]; then
+              echo $GPG_KEY | base64 --decode > signing-key
```

Comment:
Can this be extracted to a function? I can see the same code on lines 39-42

Reply:
It certainly can be extracted, but we will not save any number of lines. I didn't do that, because I don't want to touch existing release code, because there is no way to test it now.